knotx-core/knotx-knot-action/src/test/resources/fragment_form_self_out.txt

```diff
@@ -0,0 +1,8 @@
+    <form>
+        <input type="hidden" name="snippet-identifier" id="self" />
+        <div class="form-group">
+            <label for="email2">Email</label>
```

Comment:
Can we add here some handlebars code so that test proves, that nothing changes here?